examples/plot_model_card.py

```diff
@@ -72,19 +78,58 @@
 )
 license = "mit"
 
+eval_results = card.evaluate(
+    model, X_test, y_test, "r2", "random_type", "dummy_dataset", "tabular-regression"
+)
+
 card_data = CardData(
     license=license,
     tags=["tabular-classification"],
     datasets="breast-cancer",
+    eval_results=eval_results,
+    model_name="my-cool-model",
     metrics=["acc"],
 )
 
+# %% Adding metrics
+# ====================
```

Comment:
```suggestion
# %%
# Inspecting the model
# ====================
```

Please don't have the title on the same line as the `# %%` line, and also please apply to the other sections. Also, permutation importance is not a metric really.